src/py/flwr/server/strategy/fedxgb_nn_avg.py

```diff
@@ -33,7 +33,13 @@
 
 
 class FedXgbNnAvg(FedAvg):
-    """Configurable FedXgbNnAvg strategy implementation."""
+    """Configurable FedXgbNnAvg strategy implementation.
+
+    Warning
+    -------
+    This strategy will be deprecated, and move to this baseline directory
+    https://github.com/adap/flower/tree/main/baselines/hfedxgboost.
+    """
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Shoul we create sth like `flwr.common.logger.warn_experimental_feature` but for deprecated features and use it in the constructor here to print a warning message to users?